Makefile

```diff
@@ -8,32 +8,44 @@ BUILD_PATH_NDM=ndm_daemonset
 # env for specifying that we want to build node-disk-operator
 BUILD_PATH_NDO=manager
 
-# Build the node-disk-manager image.
-build: clean vet fmt shellcheck license-check-go version ndm docker_ndm ndo docker_ndo
+# Compile binaries and build docker images
+build: clean vet fmt license-check-go version ndm docker_ndm ndo docker_ndo
 
 NODE_DISK_MANAGER?=ndm
 NODE_DISK_OPERATOR?=ndo
 BUILD_PATH_NDM?=ndm_daemonset
 BUILD_PATH_NDO?=manager
 
-# Determine the arch/os
-XC_OS?= $(shell go env GOOS)
-XC_ARCH?= $(shell go env GOARCH)
-ARCH:=${XC_OS}_${XC_ARCH}
-
 # VERSION is the version of the binary.
 VERSION:=$(shell git describe --tags --always)
 
+ifeq (${XC_OS}, )
+  XC_OS:=$(shell go env GOOS)
+  export XC_OS
```

Comment:
After `XC_OS` why we need to export it again?

Reply:
These values need to be available from within the build.sh